api.js

```diff
@@ -705,11 +705,12 @@ class Api {
     return Promise.all(postParse)
   }
 
-  initContext (includeTypes) {
+  initContext (includeTypes, state) {
     const context = this.get('_context', {
       utils: this.utils,
       pathLib: this.pathLib,
-      fsLib: this.fsLib
+      fsLib: this.fsLib,
+      state: state
```

Comment:
This could just be `state`:

```suggestion
      state
```